src/Stream/Partition.cpp

```diff
@@ -82,26 +84,22 @@ bool Partition::shouldStopBasedOnPollStatus(Kafka::PollStatus CStatus) {
 void Partition::pollForMessage() {
   auto Msg = ConsumerPtr->poll();
   switch (Msg.first) {
-  case Kafka::PollStatus::Message:
-    MessagesReceived++;
-    break;
   case Kafka::PollStatus::TimedOut:
     KafkaTimeouts++;
     break;
   case Kafka::PollStatus::Error:
     KafkaErrors++;
     break;
-  case Kafka::PollStatus::EndOfPartition: // Do nothing
-    break;
-  case Kafka::PollStatus::Empty: // Do nothing
+  default:
```

Comment:
I think (though I am not sure) that _clang-tidy_ complains if not all enum possibilities are accounted for in a switch statement. I am fine with leaving this as is though.

Reply:
clang-tidy isn't complaining at me. If `default` wasn't there the compiler would warn (and thus error).